multiqc/modules/sickle/sickle.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+
+""" MultiQC module to parse output from sickle """
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+import logging
+import re
+from multiqc.plots import bargraph
+from collections import OrderedDict
+
+log = logging.getLogger(__name__)
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(name='sickle', anchor='sickle',
+                                            href="https://github.com/najoshi/sickle",
+                                            info="A windowed adaptive trimming tool for FASTQ files using quality.")
+
+        self.sickle_data = dict()
+        #parse list of log files
+        for f in self.find_log_files('sickle'):
+            self.sickle_data[f['s_name']] = self.parse_logs(f['f'])
+            self.add_data_source(f)
+        #no file found
+        if len(self.sickle_data) == 0:
+            raise UserWarning
+        self.sickle_data = self.ignore_samples(self.sickle_data)
+        self.sickle_general_stats_table()
+        self.write_data_file(self.sickle_data, 'multiqc_sickle')
+        self.read_count_plot()
+
+    def parse_logs(self, f):
+        """ Parse the Sickle standard output """
+        regexes = {
+                'reads_paired_kept': "FastQ paired records kept: ([\d,]+) .*",
+                'reads_single_kept': "FastQ single records kept: ([\d,]+).*",
+                'reads_paired_discarded': "FastQ paired records discarded: ([\d,]+) .*",
+                'reads_single_discarded': "FastQ single records discarded: ([\d,]+) .*",
+                'reads_kept': "FastQ records kept: ([\d,]+)",
+                'reads_discarded': "FastQ records discarded: ([\d,]+)"
```

Comment:
As these search patterns are specific to single-end data, I think that we can call them `reads_single_kept` / `reads_single_discarded` (but needs a minor code refactor as we can't have duplicate keys in a dict).